src/firefly/test/python/demo-notebook.ipynb

```diff
@@ -0,0 +1,423 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "fc.launchBrowser()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "#file= fc.uploadFile('./data/c.fits')\n",
+    "file= fc.uploadFile('/hydra/cm/firefly/src/firefly/test/python/data/wise-1b-1.fits')\n",
+    "#file= fc.uploadFile('../data/wise-m31-3a-1.fits')\n",
+    "#rv= fc.createRangeValuesStandard(\"LogLog\", \"Percent\",9,98)\n",
+    "#rv= fc.createRangeValuesZScale(\"Log\",80, 500,44)\n",
+    "#rv= fc.createRangeValuesZScale(\"Log\")\n",
+    "rv= fc.createRangeValuesStandard(\"LogLog\")\n",
+    "#params= {'RangeValues':rv, 'Title':\"Hello\", \"ZoomType\": \"FORCE_STANDARD\",\"InitZoomLevel\": \"2\"}\n",
+    "#params= {'RangeValues':rv, 'Title':\"Hello\"}\n",
+    "#params= {'Title':\"Hello\", \"MultiImageIdx\": \"0\" }\n",
+    "params= { 'Title':\"Hello\"}\n",
+    "\n",
+    "#status= fc.showFits(file, 'p1')\n",
+    "status= fc.showFits(file,'p1',params)\n",
+    "#status= fc.addExtension('AREA_SELECT','MyOp2','p1','MyOpExtension2')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "fc.addExtension(\"AREA_SELECT\", 'p1', 'crop2', 'second crop', imageSrc='/hydra/cm/firefly/src/firefly/html/images/icons-2014/16x16_Filter.png')\n",
+    "\n",
+    "#fc.addExtension(\"LINE_SELECT\", 'p1', 'crop2', 'second crop', imageSrc='/hydra/cm/firefly/src/firefly/html/images/icons-2014/16x16_Filter.png')\n",
+    "\n",
+    "#fc.addExtension(\"POINT\", 'p1', 'crop2', 'second crop', imageSrc='/hydra/cm/firefly/src/firefly/html/images/icons-2014/16x16_Filter.png')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "fc.zoom('p1', 5)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "fc.pan('p1', 100, 100)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "#fc.addMask(\"m1\",5,1,\"#FF00FF\",'p1', \"maskStuff\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "#fc.removeMask(\"m1\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "pParams= { 'URL' : 'http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band4.fits','ColorTable' : '9', 'title': 'hello'}\n",
+    "status= fc.showFits(fileOnServer=None, plotId='p2', additionalParams=pParams)"
```

Comment:
As noted in the diff for `showFits` in `FireflyClient.py`, it is more Pythonic to use `**kwargs` in the function definition, then make this call `status = fc.showFits(fileOnServer=None, plotId='p2', URL='http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band4.fits', ColorTable='9', title='hello')`